tests/integration/components/hyper-table-v2/manage-columns-test.ts

```diff
@@ -0,0 +1,248 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { click, fillIn, render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import { RowsFetcher, TableManager } from '@upfluence/hypertable/test-support';
+import TableHandler from '@upfluence/hypertable/core/handler';
+import sinon from 'sinon';
+import { buildColumn, buildColumnDefinition } from '@upfluence/hypertable/test-support/table-manager';
+
+const COLUMN_DEFINITIONS = [
+  { key: 'alone', category: '', clustering_key: '' },
+  { key: 'code', category: 'affiliation', clustering_key: '' },
+  { key: 'foo', category: 'influencer', clustering_key: 'instagram' },
+  { key: 'bar', category: 'influencer', clustering_key: 'youtube' }
+];
+
+const COLUMNS = [{ key: 'code', category: 'affiliation', clustering_key: '' }];
+
+module('Integration | Component | hyper-table-v2/manage-columns', function (hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(async function () {
+    this.tableManager = new TableManager();
+    this.rowsFetcher = new RowsFetcher();
+    this.handler = new TableHandler(this, this.tableManager, this.rowsFetcher);
+
+    sinon.stub(this.tableManager, 'fetchColumnDefinitions').callsFake(() => {
+      return Promise.resolve({
+        column_definitions: COLUMN_DEFINITIONS.reduce(
+          (columnDefinitions, column) => [
+            ...columnDefinitions,
+            ...[
+              buildColumnDefinition(column.key, {
+                clustering_key: column.clustering_key,
+                category: column.category
+              })
+            ]
+          ],
+          []
+        )
+      });
+    });
+
+    sinon.stub(this.tableManager, 'fetchColumns').callsFake(() => {
+      return Promise.resolve({
+        columns: COLUMNS.reduce(
+          (columns, column) => [
+            ...columns,
+            ...[
+              buildColumn(column.key, {
+                clustering_key: column.clustering_key,
+                category: column.category
+              })
+            ]
+          ],
+          []
+        )
+      });
```

Comment:
Nit (because we're in tests)
Both of the anonymous functions defined in the callsFake method are similar at 95% ; maybe move it out in a dedicated function that takes `COLUMNS` or `COLUMNS_DEFINITION` in parameter.

Again, not super important, we are in tests. 🙂